fixtures/fixtures.js

```diff
@@ -43,6 +32,14 @@ var Fixtures = {
             {"code": "SOL00000012", "updated_at": "17-05-2017 14:46.15"},
         ]
     },
+    voucherStatus: {
+        "success":
+            {"success":["RVP12345678"], "fail":[], "invalid":[]},
+        "fail":
+            {"success":[], "fail":["FAL11111111"], "invalid":[]},
+        "invalid": 
+            {"success":[], "fail":[], "invalid":["INV1"]},
```

Comment:
@katjam Ah cool thanks for the explanation! Wondered why they were inconsistent in the fixtures..